cmake/linker_script/common/common-ram.cmake

```diff
@@ -59,6 +59,8 @@ zephyr_iterable_section(NAME k_sem GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUB
 zephyr_iterable_section(NAME k_queue GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
 zephyr_iterable_section(NAME k_condvar GROUP DATA_REGION ${XIP_ALIGN_WITH_INPUT} SUBALIGN 4)
 
+
+
```

Comment:
why the extra newlines here ?
```suggestion
```

Reply:
My bad, thanks so much for correcting it. @tejlmand :)